packages/sanity/src/ui/button/Button.tsx

```diff
@@ -41,10 +58,35 @@ export const Button = forwardRef(function Button(
     size = 'default',
     mode = 'default',
     tone = 'default',
+    tooltipProps,
     ...props
-  }: ButtonProps & Omit<React.HTMLProps<HTMLButtonElement>, 'as' | 'size'>,
+  }: ButtonProps & Omit<React.HTMLProps<HTMLButtonElement>, 'as' | 'size' | 'title'>,
   ref: React.ForwardedRef<HTMLButtonElement>,
 ) {
+  if (tooltipProps) {
+    return (
+      <Tooltip
+        // Force the tooltip to render in portal
+        portal
+        {...tooltipProps}
+      >
```

Comment:
Once those tooltip changes are in Sanity UI `alpha`, let's verify if we can force all tooltips to render in portals without boundary elements and consider simplifying exposed `tooltipProps`

Reply:
Yes, I think it makes sense!